iidm/iidm-reducer/src/test/java/com/powsybl/iidm/reducer/TieLineReductionTest.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.reducer;
+
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.test.EurostagTutorialExample1Factory;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
+ */
+class TieLineReductionTest {
+
+    @Test
+    void test() {
+        var observer = new NetworkReducerObserverImpl();
+        Network network = EurostagTutorialExample1Factory.createWithTieLine();
+        var reducer = NetworkReducer.builder()
+                .withNetworkPredicate(new IdentifierNetworkPredicate(List.of("VLGEN", "VLHV1", "VLLOAD")))
```

Comment:
To improve coverage we could also add within this test a reducer with following predicates:
for a  "doubly" deleted tieline
 ```java
new IdentifierNetworkPredicate(List.of("VLHV1", "VLHV2"));
```
for a  non-deleted tielines
```java
new IdentifierNetworkPredicate(List.of("VLGEN"));
```

Reply:
for first one that would be a really fake test as it would break this simple network into part with just 2 voltage levels remaining with a load and a generator

Reply:
No, not exactly (my comment was misleading you, as I mixed up things...).
- The first predicate should lead to two voltage levels with a busbar, linked with two tielines. Without any other equipment.
- The second one is a bit strange yes as it should lead to a single voltage level with a generator. 
 
Both cases are not stranger than current unit test, which breaks the network into 2 parts with the voltage level VLLOAD left alone. And even if the unit tests are strange they are still checking that the reduction works as expected.

Reply:
Done